book/src/dev/zebra-dependencies-for-audit.md

```diff
@@ -0,0 +1,173 @@
+| [zebra-network](https://github.com/ZcashFoundation/zebra/tree/main/zebra-network) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-network) |
+| [zebra-node-services](https://github.com/ZcashFoundation/zebra/tree/main/zebra-node-services) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-node-services)  |
+| [zebra-rpc](https://github.com/ZcashFoundation/zebra/tree/main/zebra-rpc) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-rpc) |
+| [zebra-script](https://github.com/ZcashFoundation/zebra/tree/main/zebra-script) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-script) |
+| [zebra-state](https://github.com/ZcashFoundation/zebra/tree/main/zebra-state) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-state) |
+| [zebrad](https://github.com/ZcashFoundation/zebra/tree/main/zebrad) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebrad) |
+
+### Zcash/ZF dependencies
+
+| Name | Version | Notes
+|------| ------- | -----
+| [ed25519-zebra](https://github.com/ZcashFoundation/ed25519-zebra) | [3.1.0](https://github.com/ZcashFoundation/ed25519-zebra/releases/tag/3.1.0) |
+
+---
+## Partial Audit 
+
+### Crates
+
+| Name | Version | Notes
+|------| ------- | -----
+| [zebra-utils](https://github.com/ZcashFoundation/zebra/tree/main/zebra-utils) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-utils) | <i>Only [zebra-checkpoints](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-utils/src/bin/zebra-checkpoints) utility needs to be audited.</i>
+
+### Zcash/ZF dependencies
+
+| Name | Version | Audited | Notes
+|------| --------|-------- | -----
+| [zcash_proofs](https://github.com/zcash/librustzcash/tree/zcash_proofs-0.8.0) | [0.8.0](https://github.com/zcash/librustzcash/tree/zcash_proofs-0.8.0) | [qedit](https://hackmd.io/@qedit/zcash-nu5-audit) | <i>Only the proof parameter download code in [downloadreader.rs](https://github.com/zcash/librustzcash/blob/zcash_proofs-0.8.0/zcash_proofs/src/downloadreader.rs), [hashreader.rs](https://github.com/zcash/librustzcash/blob/zcash_proofs-0.8.0/zcash_proofs/src/hashreader.rs), and [lib.rs](https://github.com/zcash/librustzcash/blob/zcash_proofs-0.8.0/zcash_proofs/src/lib.rs) need to be audited.</i>  |
+| [zcash_script](https://github.com/ZcashFoundation/zcash_script) | [0.1.7](https://github.com/ZcashFoundation/zcash_script/releases/tag/v0.1.7) || <i>The C++ parts of `zcashd` got audited as part of the ECC audit, so we only need to audit [1](https://github.com/zcash/zcash/blob/master/src/script/zcash_script.cpp),[2](https://github.com/zcash/zcash/blob/master/src/script/zcash_script.h) and the rust code in [3](https://github.com/ZcashFoundation/zcash_script/tree/master/src)</i>
+
+---
+## Out of Scope 
```

Comment:
We've updated some of these dependency versions since this list was generated.
Do you think this note is enough?

(I'm not sure if it's worth regenerating the table with the correct versions.)

```suggestion
## Out of Scope

All versions of these dependencies are out of scope for the audit.
(Some of the dependency versions in these tables might be outdated.)

The latest versions of Zebra's dependencies are in [`Cargo.lock`](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.1/Cargo.lock), including transitive dependencies.
```

Reply:
Versions for out-of-scope crates don't seem too pertinent so this note should be enough.